### ISSUE TITLE
Gulp search no longer returns results

### ISSUE THREAD

**Issue by jeffbyrnes** [willfarrell/alfred-pkgman-workflow#68]
Found while checking out #63.

It seems that the [Gulp.js registry](http://gulpjs.com/plugins/) has changed slightly, and is no longer being searched by the workflow.

Looking at the code for that search, it seems that page is now querying a (mirror?) Node.js repository with a set of keywords to filter for Gulp stuff:

```js
return $http.get("http://registry.gulpjs.com/_search", {
    params: {
        q: "keywords:gulpplugin,gulpfriendly",
        fields: fields.join(","),
        from: start,
        size: size
    },
    transformResponse: $http.defaults.transformResponse.concat([formatData])
})
```

We’ll need to tweak the cache that’s created for Gulp (or, perhaps, stop using said cache).

**Comment by itadakimas** [comment]
+1

**Comment by Link2k5** [comment]
+1

**Comment by keeprock** [comment]
+1

**Comment by willfarrell** [comment]
Don't worry, I hear you guys. I will carve out some time this weekend to look at this.

**Comment by jeffbyrnes** [comment]
@willfarrell sorry I haven’t been able to help much lately. Overwhelmed with day job work!

**Comment by willfarrell** [comment]
@jeffbyrnes no worries.

Fixed in v3.05, let me know if there are any bugs.
